scipy/linalg/flapack_gen.pyf.src

```diff
@@ -1371,3 +1371,41 @@ subroutine <prefix2c>ggev(compute_vl,compute_vr,n,a,b,alpha,beta,vl,ldvl,vr,ldvr
     integer intent(out):: info
 
 end subroutine <prefix2>ggev
+
+subroutine <prefix>geequ(m,n,a,lda,r,c,rowcnd,colcnd,amax,info)
+
+    callstatement {(*f2py_func)(&m,&n,a,&lda,r,c,&rowcnd,&colcnd,&amax,&info);}
+    callprotoargument int*,int*,<ctype>*,int*,<ctypereal>*,<ctypereal>*,<ctypereal>*,<ctypereal>*,<ctypereal>*,int*
+
+    integer intent(hide),depend(a) :: m = shape(a,0)
+    integer intent(hide),depend(a) :: n = shape(a,1)
+    integer intent(hide),depend(a) :: lda = MAX(1,shape(a,0))
+
+    <ftype> intent(in), dimension(m,n) :: a
+    <ftypereal> intent(out),dimension(m),depend(m) :: r
+    <ftypereal> intent(out),dimension(n),depend(n) :: c
```

Comment:
depend(a, m)?

Reply:
That's resolved via f2py. I think I've found why the segfaulting happens. This should fix the faults but the variability would still be there across Travis and other CIs unfortunately. I'll try to find some other example that would make the tests fail uniformly.

Reply:
IIRC f2py was in some cases not resolving transitive dependencies for some python/numpy versions. Glad if this have improved in the meantime